editor/src/clj/editor/app_view.clj

```diff
@@ -1405,9 +1406,20 @@
       (let [main-scene (.getScene ^Stage main-stage)
             render-build-error! (make-render-build-error main-scene tool-tab-pane build-errors-view)]
         (build-errors-view/clear-build-errors build-errors-view)
-        (if (debug-view/can-attach? prefs)
-          (attach-debugger! workspace project prefs debug-view render-build-error!)
-          (run-with-debugger! workspace project prefs debug-view render-build-error! web-server))))))
+        (try
+          (if (debug-view/can-attach? prefs)
+            (attach-debugger! workspace project prefs debug-view render-build-error!)
+            (run-with-debugger! workspace project prefs debug-view render-build-error! web-server))
+          (catch Exception e
```

Comment:
It might better to only catch the specific type of exception thrown in the situation we want to catch?